certora/specs/ghoVariableDebtToken.spec

```diff
@@ -128,6 +130,17 @@ invariant discountCantExceed100Percent(address user)
 			require(indexAtTimestamp(e.block.timestamp) >= ray());
 		}
 	}
+    
+/**
+* @title at any point in time, the user's discount rate isn't larger than DISCOUNT_RATE
+**/
+invariant discountCantExceedDiscountRate(address user)
+	getUserDiscountRate(user) <= discStrategy.DISCOUNT_RATE()
+	{
+		preserved updateDiscountDistribution(address sender,address recipient,uint256 senderDiscountTokenBalance,uint256 recipientDiscountTokenBalance,uint256 amount) with (env e) {
+			require(indexAtTimestamp(e.block.timestamp) >= ray());
```

Comment:
function indexAtTimestamp has already the same require. This require should be redundant. Does the invariant fail without it?

Reply:
removed